### PR TITLE
feat(wallet): More explicit feedback when sending coins is not possible

### PR DESCRIPTION
**Pull request by SylEze**
Different feedbacks:

- Warn message when user has not enough funds, thus the `Send` button is disabled
- Warn message when user has not enabled the coin of the fees - in this case, clicking on `Send` button will prompt a modal to enable the fees coin
- User has no funds to pay the fees, thus the `Send` button is disabled

Related to https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/880

### CODE REVIEW

**Comment by Milerius** [comment]
> Different feedbacks:
> 
> * Warn message when user has not enough funds, thus the `Send` button is disabled
> * Warn message when user has not enabled the coin of the fees - in this case, clicking on `Send` button will prompt a modal to enable the fees coin
> * User has no funds to pay the fees, thus the `Send` button is disabled
> 
> Related to #880

At first glance is not working as expected, i got the warning correctly that i have to enabled BNB, but once i enable it the warning persist and i cannot do anything else.

**Review by Milerius — CHANGES_REQUESTED** [review]
check the comment above

**Comment by SylEze** [comment]
> > Different feedbacks:
> > 
> > * Warn message when user has not enough funds, thus the `Send` button is disabled
> > * Warn message when user has not enabled the coin of the fees - in this case, clicking on `Send` button will prompt a modal to enable the fees coin
> > * User has no funds to pay the fees, thus the `Send` button is disabled
> > 
> > Related to #880
> 
> At first glance is not working as expected, i got the warning correctly that i have to enabled BNB, but once i enable it the warning persist and i cannot do anything else.
> 
> image
> 
> State need to be recalculated when the size of the portfolio changed

This is something I've tested, and as shown in the following recording it's working on my side:

https://user-images.githubusercontent.com/14373103/117436183-fd6bf800-af2e-11eb-9a20-46b7322a0ba3.mov

**Review by Milerius — DISMISSED** [review]

**Comment by Milerius** [comment]
works for me now, waiting for @smk762 approval too !

File: src/core/atomicdex/pages/qt.wallet.page.cpp
Comment (smk762) [inline src/core/atomicdex/pages/qt.wallet.page.cpp:50]: Spelling mistake: Should be "funds"

**Comment by smk762** [comment]
After doing a "send max" transaction, "send" button remains active.
After receiving funds from faucet when balance was previously zero, "send" button remains inactive.
image

App requires restart for button to activate/deactivate.

Confirm send disabled on ERC-20 token when no ETH present.
"parent chain balance is zero" error message present when attempting to trade ERC-20 with no ETH to cover gas.

**Comment by SylEze** [comment]
> After doing a "send max" transaction, "send" button remains active.
> After receiving funds from faucet when balance was previously zero, "send" button remains inactive.
> image
> 
> App requires restart for button to activate/deactivate.
> 
> Confirmed send disabled on ERC-20 token when no ETH present.
> "parent chain balance is zero" error message present when attempting to trade ERC-20 with no ETH to cover gas.
> Confirmed "enable ETH" popup appears when not activated and attempting to send ERC-20.

@smk762 It should be fixed now

**Pull request merged**
